docs/getting-started/cash-app-pay.md

```diff
@@ -24,46 +24,46 @@ NEW (v4.3.0)
 {: .alert }
 > Cash App Pay is currently available in the following region(s): US
 
-Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App PayKit SDK.
+Cash App Pay support can be added using your Afterpay merchant account. This requires a token to be generated by a server-to-server call to the Afterpay API Create Checkout endpoint with a parameter of `isCashAppPay` set to `true`. This method requires importing and implementing the Cash App Pay Kit SDK.
 
 {: .info }
 > When creating a checkout token, both `redirectConfirmUrl` and `redirectCancelUrl` must be set. Failing to do so will cause undefined behavior. The SDK’s example merchant server sets the parameters [here][example-server-props]{:target='_blank'}. See more by checking the [api reference][api-reference-props]{:target='_blank'}.
```

Comment:
```suggestion
> When creating a checkout token, you must set both `redirectConfirmUrl` and `redirectCancelUrl`. If they are not set, it will cause undefined behavior<!--can you be more specific on "undefined behavior"-->. The SDK’s example merchant server sets the parameters [here][example-server-props]{:target='_blank'}. See more details at [api reference][api-reference-props]{:target='_blank'}.
```

Reply:
Addressed in [3f0282c](https://github.com/afterpay/sdk-android/pull/185/commits/3f0282c492fe293fd098fd2f2b4b0912de2acf1e)

Reply:
@ScottAntonacAP Can you change the last line to See more details at [Redirect Method]([api-reference-props]{:target='_blank'} in the Standard Checkout API.

Reply:
Addressed in [522aa7b](https://github.com/afterpay/sdk-android/pull/185/commits/522aa7b27c98d5b78b5cc00a438de93adecf5e74)